package.json

```diff
@@ -66,30 +66,30 @@
   },
   "homepage": "https://github.com/ReactiveX/RxJS",
   "devDependencies": {
-    "babel": "^5.8.23",
-    "babelify": "^6.1.2",
-    "benchmark": "^1.0.0",
+    "babel": "5.8.23",
+    "babelify": "6.1.2",
+    "benchmark": "1.0.0",
     "benchpress": "2.0.0-alpha.37.2",
-    "browserify": "^11.0.0",
-    "colors": "^1.1.2",
-    "coveralls": "^2.11.4",
-    "esdoc": "^0.2.5",
+    "browserify": "11.0.0",
+    "colors": "1.1.2",
+    "coveralls": "2.11.4",
+    "esdoc": "0.2.5",
     "eslint": "1.5.1",
-    "fs-extra": "^0.24.0",
-    "glob": "^5.0.14",
-    "google-closure-compiler": "^20150920.0.0",
-    "http-server": "^0.8.0",
-    "istanbul": "^0.3.22",
-    "jasmine": "^2.3.2",
-    "jasmine-core": "^2.2.0",
-    "lodash": "^3.5.0",
-    "platform": "^1.3.0",
-    "promise": "^7.0.3",
+    "fs-extra": "0.24.0",
+    "glob": "5.0.14",
+    "google-closure-compiler": "20150920.0.0",
+    "http-server": "0.8.0",
+    "istanbul": "0.3.22",
+    "jasmine": "2.3.2",
+    "jasmine-core": "2.3.4",
+    "lodash": "3.5.0",
+    "platform": "1.3.0",
+    "promise": "7.0.3",
     "protractor": "2.2.0",
     "rx": "latest",
-    "tslint": "^2.5.0",
-    "typescript": "^1.7.0-dev.20150901",
-    "watch": "^0.16.0"
+    "tslint": "2.5.0",
+    "typescript": "1.6.2",
```

Comment:
Think do not need beta of tsc 1.7 yet, lowered to latest public. Once this PR(https://github.com/Microsoft/TypeScript/pull/5290) for TS side merging support is ready, can bump up to use those specific version later.